.github/workflows/build-master.yml

```diff
@@ -0,0 +1,26 @@
+name: Build master branch of Ballerina datamapper compiler extension
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
```

Comment:
BALLERINA_STDLIB_TOKEN is now depreciated, check latest stdlibs workflows for updated token

Reply:
Updated to match the latest tokens